src/devices/Pn532/samples/Program.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Pn532;
```

Comment:
are all of these used? it seems quite a long list of namespace and since this is sample I'd rather reduce if possible

Reply:
Great catch, cleaned.